### ISSUE TITLE
AsciidocFx 1.6.9: slides with revealjs or deckjs backend are under each other

### ISSUE THREAD

**Issue by amekki** [asciidocfx/AsciidocFX#396]
Hi,
I created a slide (new -> slide) and then clicked on the html button to convert it. But when open with chrome, slides are not well rendered. They are under each other like a printable version.
Could you check please ?

**Comment by albertoSoto** [comment]
I updated too to 1.6.9 and all time crashes showing "extjs can't not be loaded. but don't worry"

**Comment by rahmanusta** [comment]
That weird message will be removed in next release. Any other issue?

**Comment by albertoSoto** [comment]
Yes, Slides module does not work at all rendering reveal or deck after upgrading to 1.6.9. I have deleted all registry entries and tried over 1.6.6 and it works again. Browser component doesn't start. 
Could be some issues with perms on upgrade.

By now I can keep working with 1.6.6
